audit.go

```diff
@@ -302,6 +310,39 @@ func createFilters(config *viper.Viper) []AuditFilter {
 	return filters
 }
 
+func logStart(c *viper.Viper, w *AuditWriter) error {
+	f, err := os.Open(c.ConfigFileUsed())
+	if err != nil {
+		return err
+	}
+	defer f.Close()
+
+	s := sha256.New()
+	if _, err := io.Copy(s, f); err != nil {
+		return err
+	}
+
+	b, err := ioutil.ReadFile("/proc/uptime")
+	if err != nil {
+		return err
+	}
+	fields := strings.Split(string(b), " ")
+	uptime, err := strconv.ParseFloat(fields[0], 64)
+
+	if err != nil {
+		return err
+	}
+
+	w.Write(&m{
```

Comment:
This returns an error which should be handled: https://github.com/slackhq/go-audit/blob/ebcc4b5596e71716a32e08f1671aa278fede5be9/writer.go#L23